public/app/features/dashboard-scene/scene/layout-rows/RowsLayoutManager.tsx

```diff
@@ -186,9 +187,13 @@ export class RowsLayoutManager extends SceneObjectBase<RowsLayoutManagerState> i
   }
 
   public static createFromLayout(layout: DashboardLayoutManager): RowsLayoutManager {
-    let rows: RowItem[];
+    let rows: RowItem[] = [];
 
-    if (layout instanceof DefaultGridLayoutManager) {
+    if (layout instanceof TabsLayoutManager) {
+      for (const tab of layout.state.tabs) {
+        rows.push(new RowItem({ layout: tab.state.layout, title: tab.state.title }));
```

Comment:
This is leading to nested rows if a tab layout has rows inside. I think we need to check the layout type of each tab.

Reply:
@bfmatei I think nested rows and nested tabs should be allowed, no easy way to prevent in the schema, so we need to support it. 

I think it would be counter intuitive when switching from tabs to rows layout if we changed the layout of the tabs content